contact.css

```diff
@@ -0,0 +1,225 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:300:400');
+
+*, *:before, *:after {
+	box-sizing: border-box;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+  }
+  
+  body {
+	background: linear-gradient(
+		60deg,
+		rgba(84, 58, 183, 1) 0%,
+		rgba(0, 172, 193, 1) 100%);
+	font-size: 12px;
```

Comment:
- Try to avoid use of px use rem or em instead that will make the website more responsive

Reply:
Please review now

Reply:
Please review now changes done.